keras_cv/layers/preprocessing/base_image_augmentation_layer.py

```diff
@@ -130,11 +130,13 @@ def augment_image(self, image, transformation):
     """
 
     def __init__(self, seed=None, **kwargs):
-        force_generator = kwargs.pop("force_generator", False)
-        self._random_generator = keras_backend.RandomGenerator(
-            seed=seed, force_generator=force_generator
-        )
+        if seed:
```

Comment:
maybe better to do the explicit `if seed is not None` check here, avoid unexpected cases of people passing strange seed input and getting the global generator behavior

Reply:
That makes sense. Done